pootle/apps/accounts/management/commands/update_user_email.py

```diff
@@ -20,13 +20,21 @@
 
 
 class Command(UserCommand):
-    args = "user email"
     help = "Update user email address"
 
-    def handle(self, *args, **kwargs):
-        super(Command, self).handle(*args, **kwargs)
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "user",
+        )
+        parser.add_argument(
+            "email"
```

Comment:
Seems adding them messes with expected output in errors, so taking it away. You're still limited to one anyway.